notifications/push/fcm/notification.go

```diff
@@ -25,30 +26,35 @@ func NewNotification(key string) NotificationConstructor {
 		client := fcm.NewFcmClient(key).
 			SetDelayWhileIdle(true).
 			SetContentAvailable(true).
+			SetPriority(fcm.Priority_HIGH).
 			SetTimeToLive(fcm.MAX_TTL)
 
 		return &Notification{client}
 	}
 }
 
-// Send send to the tokens list.
-func (n *Notification) Send(body string, payload fcm.NotificationPayload, tokens ...string) error {
-	data := map[string]string{
-		"msg": body,
-	}
+// Send sends a push notification to the tokens list.
+func (n *Notification) Send(dataPayloadJSON string, tokens ...string) error {
+	log.Debug("calling Firebase Send", "dataPayloadJSON", dataPayloadJSON)
```

Comment:
i like the payload name more, it is obvious what it is, not sure if JSON part is that important

Reply:
The JSON in the name is to inform the callers that the string will be interpreted (unmarshalled) as a JSON object.